app/simformat.hs

```diff
@@ -64,6 +66,10 @@ parseOperation =
       <> Opt.metavar "TARGET"
       <> Opt.help "operate on a file in-place instead of reading from stdin and writing to stdout"
     repo = pure Repo
+    editor = Opt.flag' Editor
+      $ Opt.long "editor"
+      <> Opt.short 'e'
+      <> Opt.help "read from stdin; output to stdout. For use with per-file editor integration"
```

Comment:
One thing I notice here is that there's a period but none of the help start with a capital letter. Do with it what you will. Could capitalize each help or refactor this into a single sentence or do nothing and I'll fix it.